cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/detector/SlowBrokerFinderTest.java

```diff
@@ -25,21 +25,28 @@
 
 
 public class SlowBrokerFinderTest {
-  private final static double NORMAL_BYTES_IN_RATE = 1024.0 * 1024.0;
-  private final static double SMALL_BYTES_IN_RATE = 1024.0;
+  private final static double NORMAL_BYTES_IN_RATE = 4096;
+  private final static double BYTE_IN_RATE_VARIANCE_PERCENTILE = 0.1;
   private final static double NORMAL_LOG_FLUSH_TIME_MS = 100.0;
+  private final static double LOG_FLUSH_TIME_MS_VARIANCE_PERCENTILE = 0.25;
```

Comment:
What is variance percentile? Can we rename it to better represent what it refers to?